sys-fs/tmsu/tmsu-0.6.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_PN="github.com/oniony/TMSU/"
+DESCRIPTION="Files tagger and virtual tag-based filesystem"
+HOMEPAGE="https://github.com/oniony/TMSU/wiki"
+SRC_URI="https://github.com/oniony/TMSU/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
I haven't looked at this package, but I suspect that they are build time dependencies so should go in DEPEND rather than RDEPEND. If you want to verify, a quick way to do this would be to emerge the package and remove the dependencies after the fact and see if it still runs.